### ISSUE TITLE
animations:enabled=false is ignored during startup since v0.25.0

### ISSUE THREAD

**Issue by jbeich** [hyprwm/Hyprland#2775]
### Hyprland Version

v0.27.2-32-g375e8385

### Bug or Regression?

Regression

### Description

Regressed by ddfeebad3dea. When Hyprland is started the applications run immediately on startup have zoom out animation even if disabled in config. This can be distracting and interfere with benchmarking or nesting under another compositor. For example, parent Hyprland may *not* want child Hyprland instances to animate fullscreen content within.

Workaround: prepend `sleep 2 && ` to `exec` or `exec-once` to add minor delay

### How to reproduce

```
$ cat /tmp/hyprland.conf
animations:enabled = false
exec-once = glmark2-wayland -b build && hyprctl dispatch exit 0

$ Hyprland -c /tmp/hyprland.conf
```

**Comment by jbeich** [comment]
Oops, shouldn't have used `--fullscreen` in steps to reproduce as it makes it harder to notice how window borders unstretch.

**Comment by vaxerski** [comment]
fixed in 1a7fb1572ad5fc3bd95b0fa366faee68f8622acf
